SUPPORT.md

```diff
@@ -0,0 +1,22 @@
+# Support for deploying and using KEDA
+
+Welcome to KEDA! We use GitHub for tracking bugs and feature requests but here are some pointers to get help.
```

Comment:
We should be more vocal, that the support is "best effort only", it is not a commercial support or something like this, that could be expected by someone.

Reply:
Agreed, I will.

Reply:
PTAL